lib/crypto/crypto_keys_subr.c

```diff
@@ -208,14 +209,8 @@ crypto_keys_subr_import_RSA_priv(RSA ** key, const uint8_t * buf, size_t buflen)
 		goto err2;
 
 	/* Load values into the RSA key. */
-	(*key)->n = n;
-	(*key)->e = e;
-	(*key)->d = d;
-	(*key)->p = p;
-	(*key)->q = q;
-	(*key)->dmp1 = dmp1;
-	(*key)->dmq1 = dmq1;
-	(*key)->iqmp = iqmp;
+	if (crypto_compat_RSA_import(key, n, e, d, p, q, dmp1, dmq1, iqmp))
+		goto err1;
```

Comment:
Incidently, adding this `err1` fixes a compile warning in current Tarsnap master: At the moment, we skip from err0 to err2 and never use err1, which causes clang to complain about an unused label.

I'm choosing to claim that I intentionally wrote the commits this way, since it minimizes the diffs.  That said, in the future I'll make sure that such things are mentioned in the commit messages, are resolved within a single PR, and I'll enable stricter checking in `.travis.yml`.